session.go

```diff
@@ -576,7 +576,43 @@ func (s *ProxySession) proxyFilterAppToTor() {
 						}
 						continue
 					} else {
-						log.Noticef("allowed ADD_ONION with %s", cmdLine)
+						if s.policy.OzForwardOnion == true {
+							if s.policy.OzApp == "" {
+								log.Errorf("Missing Oz profile name, filter policy syntax error on %s so DENY: %s A->T: [%s]\n", s.policy.ExecPath, appName, cmdLine)
+								_, err = s.appConnWrite(false, []byte("250 Tor Control command proxy denied: filtration policy.\r\n"))
```

Comment:
should the protocol return code be 250 here? or should we change it to 510 like the other deny messages?